scene/gui/line_edit.cpp

```diff
@@ -2280,7 +2276,13 @@ void LineEdit::_shape() {
 	if (text.length() == 0 && ime_text.length() == 0) {
 		t = placeholder_translated;
 	} else if (pass) {
-		t = secret_character.repeat(text.length() + ime_text.length());
+		// Allow secret_character as empty strings, act like if a space was used as a secret character.
+		String secret = " ";
+		// Allow values longer than 1 character in the property, but trim characters after the first one.
```

Comment:
One question for evaluation is the usability of multi codepoint entries for non-latin scripts, but would require some integration with the text server, so out of scope for this, but something to consider

Reply:
Yes, the origin code will assert if `p_string.length() != 1` so it assumes it must be lation scripts like English. I think maybe add support for non-latin secret character should be a standalone proposal.

Reply:
Agreed, might be worth adding a `TODO` there for it